code-orange/AppDelegate.swift

```diff
@@ -41,12 +42,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     UIApplication.shared.applicationIconBadgeNumber = 0
 
+    UIApplication.shared.setMinimumBackgroundFetchInterval(60.minutes)
+
     if !locationService.isUpdatingLocation {
       locationService.startUpdatingLocation()
     }
 
     return true
   }
+
+  func application(_ application: UIApplication,
+                   performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+    // TODO: Manage differential data fetch and call the completion handler correspondingly
+    communicator.updateInfectedLocations()
+    completionHandler(.newData)
```

Comment:
if we answer with newData always at some point Apple may give us less running intervals overall